### PR TITLE
Added helper method as mentioned in issue 73

### PR DESCRIPTION
**Pull request by jshinn6788**
thanks for pointing out getModel.  Here is a helper method so this issue doesn't get raised again.

### CODE REVIEW

File: src/main/java/com/aventstack/extentreports/ExtentTest.java [hunk 1]
```diff
@@ -1201,7 +1201,16 @@ public Status getStatus() {
...
     }
-    
+
+    /**
+     * Convenience method for setting the description of the internal test model
+     *
+     * @param desc The description of the test case in the logs
+     */
+    public void setDescription(String desc) {
+        test.setDescription(desc);
```
Comment (virenv) [inline src/main/java/com/aventstack/extentreports/ExtentTest.java]: I would suggest we add a test here to check this method actually updates the Description.

Comment (virenv) [inline src/main/java/com/aventstack/extentreports/ExtentTest.java]: Can you do one thing here, please add some logic to test if the string is empty, null or trim it from preceding and proceeding spaces.
It will be better if you go ahead and move that logic inside the setDescription method instead of doing it on the caller side.

**Comment by virenv** [comment]
@jshinn6788 : Marking you just to make sure that you have the updates on your PR. Any estimated time for this PR that you can think of?

**Comment by jshinn6788** [comment]
@virenv the changes requested are a bit more than what I was thinking.  Therefore, I will go ahead and just close this PR.  However, I would like to see what you all come up with as a final resolution.

Thank you,
Jess

**Comment by virenv** [comment]
@jshinn6788 : The changes are not that much, please try it out and let us know when you need help. What do you say?
